foo.txt

```diff
@@ -0,0 +1,5 @@
+Added somestuff
+with some more stuff
+
+and then some
```

Comment:
Here's an inline comment. Push a new version and let's see if the inline survives